cluster/configs/src/main/java/com/linkedin/openhouse/cluster/configs/TblPropsToggleRegistry.java

```diff
@@ -0,0 +1,31 @@
+package com.linkedin.openhouse.cluster.configs;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+/** A central registry for all table properties that are managed through Feature Toggle. */
+@Component
+public class TblPropsToggleRegistry {
+
+  public static final String ENABLE_TBLTYPE = "enable_tabletype";
+
+  // TODO: Using these vocabularies as MySQL validation
+  private final Map<String, String> featureKeys = new HashMap<>();
+
+  @PostConstruct
+  public void initializeKeys() {
+    // placeholders: demo purpose
+    featureKeys.put("openhouse.tableType", ENABLE_TBLTYPE);
```

Comment:
this is test code sneaking into prod? is there a better way to do this? maybe make the featurekeys protected visiblefortesting?

Reply:
I can just remove this code.